.jenkins/cscs-perftests/comment_github.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash -l
+set -eux
+
+echo $PWD
+pushd perftests-reports/reports-comparison
+
+curl \
+  -X POST \
+  -H "Authorization: token ${GITHUB_TOKEN}" \
```

Comment:
```suggestion
  -H "Authorization: token ${STELLARBOT_GITHUB_TOKEN}" \
```
That should make it start commenting with `StellarBot` instead of my account... I'll delete my token once this is working.

Reply:
it's already the stellarbot token normally cause you set the environment `GITHUB_TOKEN` here:
https://github.com/STEllAR-GROUP/hpx/blob/master/.jenkins/cscs-perftests/Jenkinsfile#L27

Reply:
Apparently the error is about an invalid JSON, I'm investigating further..

Reply:
> it's already the stellarbot token normally cause you set the environment `GITHUB_TOKEN` here:
> https://github.com/STEllAR-GROUP/hpx/blob/master/.jenkins/cscs-perftests/Jenkinsfile#L27

D'oh, indeed. I'll go hide.